fio-prepare-jobfile

```diff
@@ -1,28 +1,49 @@
 #!/bin/bash
+# -*- mode: sh; indent-tabs-mode: nil; sh-basic-offset: 4 -*-
+# vim: autoindent tabstop=4 shiftwidth=4 expandtab softtabstop=4 filetype=bash
 
 # Just copy/rename the jobfile specified in the params to the default location
 # or generate a new one.
 #
 # Expected outcome is a "fio.job" file in the CWD
 
+jobfile=""
+
 while [ ! -z $1 ]; do
-	# TODO: support --arg val
-	arg=`echo $1 | awk -F= '{print $1}'`
-	if [ "$arg" == "--jobfile" ]; then
-		jobfile_path="`echo $1 | awk -F= '{print $2}'`"
-	else
-		echo "Ignoring this argument: $arg"
-	fi
-	shift
+    # TODO: support --arg val
+    arg=`echo $1 | awk -F= '{print $1}'`
+    if [ "$arg" == "--jobfile" ]; then
+        jobfile="`echo $1 | awk -F= '{print $2}'`"
+	echo "Found jobfile argument = ${jobfile}"
+    else
+        echo "Ignoring this argument: $arg"
+    fi
+    shift
 done
+
+jobfile_path=""
+if [ -n "${jobfile}" ]; then
+    if [ -n "${CRUCIBLE_HOSTFS_PWD}" ]; then
+        echo "Using CRUCIBLE_HOSTFS_PWD=${CRUCIBLE_HOSTFS_PWD}"
+        jobfile_path+="/hostfs"
+        jobfile_path+=${CRUCIBLE_HOSTFS_PWD}
+        jobfile_path+="/"
+    fi
+    jobfile_path+=${jobfile}
+fi
+
 if [ -z "$jobfile_path" ]; then
-	echo [global] >fio.job
-	echo [job-1] >>fio.job
-elif [ "$jobfile_path" == "EMPTY_JOB_FILE" ]; then
-	echo >fio.job
+    echo "Creating simple fio.job:"
+    echo [global] >fio.job
+    echo [job-1] >>fio.job
+    cat fio.job
+elif [ "$jobfile" == "EMPTY_JOB_FILE" ]; then
+    echo "Creating empty fio.job"
+    echo >fio.job
```

Comment:
touch fio.job

Reply:
I'm probably overly pessimistic, but the point of this is to create an empty file and touch won't do that if the file already exists for some reason (it shouldn't though...).